src/Groups/GAPGroups.jl

```diff
@@ -241,16 +241,16 @@ end
 # need this function just for the iterator
 Base.length(x::GAPGroup)::Int = order(x)
 
-"""
-    perm(L::AbstractVector{<:Integer})
-Return the permutation `x` sending `i` into `L[i]` for every `i`. `L` must contain exactly one time every integer from 1 to n for n = length(`L`).
-The parent of `x` is set as Sym(n), where n is the largest moved point of `x`.
-"""
 # FIXME: clashes with AbstractAlgebra.perm method
 #function perm(L::AbstractVector{<:Base.Integer})
 #   return PermGroupElem(symmetric_group(length(L)), GAP.Globals.PermList(GAP.julia_to_gap(L)))
 #end
 # FIXME: use name gap_perm for now
+"""
+    gap_perm(L::AbstractVector{<:Integer})
+Return the permutation `x` sending `i` into `L[i]` for every `i`. `L` must contain exactly one time every integer from 1 to `n` for `n` = length(`L`).
```

Comment:
```suggestion
Return the permutation `x` which maps every `i` from `1` to `length(L)` to `L[i]`. `L` must contain every integer from 1 to `length(L)` exactly, otherwise an exception is thrown.
```